src/vm.rs

```diff
@@ -22,6 +22,9 @@ impl VM {
     }
 
     pub fn instruction_cycle(&mut self) {
+        self.registers.set(Register::PC, 0x3000);
```

Comment:
You could use a constant like `PC_START` for `0x3000`.

Reply:
True!